streaming_jupyter_integrations/magics.py

```diff
@@ -379,15 +365,34 @@ def __flink_execute_sql_file(self, path: Union[str, os.PathLike[str]]) -> None:
         finally:
             self.background_execution_in_progress = False
 
-    @staticmethod
-    def __load_plugins() -> None:
+    def __extend_classpath(self, classpath_to_add: str) -> None:
+        pipeline_classpaths = "pipeline.classpaths"
+        current_classpaths = (
+            self.st_env.get_config()
+                .get_configuration()
+                .get_string(pipeline_classpaths, "")
+        )
+        new_classpath = (
+            f"{current_classpaths};{classpath_to_add}"
+            if len(current_classpaths) > 0
+            else classpath_to_add
+        )
```

Comment:
```suggestion
        new_classpath = ';'.join([current_classpaths, classpath_to_add])
```
? But I'm not against the current code, it's just a suggestion :)

Reply:
Ok, it won't work, you're right. So nevermind.